contracts/V1/StateChannelDiamondProxy/AStateChannelManagerProxy.sol

```diff
@@ -182,15 +185,22 @@ abstract contract AStateChannelManagerProxy is
      */
     function postBlockCalldata(SignedBlock memory signedBlock, uint maxTimestamp) public override {
         //Time is the only race condition we need to take into account
-        require(block.timestamp <= maxTimestamp, ErrorBlockCalldataTimestampTooLate());
+        
+        if (block.timestamp > maxTimestamp) {
+           revert ErrorBlockCalldataTimestampTooLate();
+        }
```

Comment:
why change the 'require' to 'if -> revert'

Reply:
So, require is supoosed to return a string error not custom error,

Reply:
https://soliditylang.org/blog/2024/05/21/solidity-0.8.26-release-announcement/

I'm probably using the latest compiler since on my end it compiles without errors